ci/docker-compose.yml

```diff
@@ -10,7 +10,7 @@ services:
       POSTGRES_PASSWORD: ''
 
   mysql:
-    image: mariadb:10.2
+    image: mariadb:10.4.12
```

Comment:
do we actually need this new? iow are we now not supporting an older version?

generally we should not touch anything in the CI except for a dedicated PR

this way we don’t regress 

also we want to test oldest versions that are supported and not newest